test/TestContour.cc

```diff
@@ -127,43 +123,43 @@ class ContourTest : public ::testing::Test {
 };
 
 TEST_F(ContourTest, NoSmoothingFitsFile) {
-    GenerateContour(500, 500, IMAGE_OPTS, CARTA::FileType::FITS, CARTA::SmoothingMode::NoSmoothing);
+    GenerateContour("500_500_image_opts.fits", CARTA::FileType::FITS, CARTA::SmoothingMode::NoSmoothing);
```

Comment:
In all these tests you should construct the path directly and pass it into the function. E.g.
```
GenerateContour(FitsImages() / "500_500_image_opts.fits", CARTA::SmoothingMode::NoSmoothing);
```